Update step 5 for configuring Azure AD in Rancher (#441)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

From the Rancher UI, enter information about your AD instance hosted in Azure to complete configuration.

1. Log into Rancher.

1.	In the top left corner, click **☰ > Users & Authentication**.

1. In the left navigation menu, click **Auth Provider**.

1. Click **AzureAD**.

1. Complete the **Configure Azure AD Account** form using the information you copied while completing [Copy Azure Application Data](#4-copy-azure-application-data).

    :::caution 
    

>     The Azure AD user credentials entered in this step will be mapped to the Rancher local principal account and assigned administrator privileges. Make sure you are comfortable giving administrator privileges to this Azure AD account.

Reviewer: ```suggestion
    The Azure AD account will be granted administrator privileges, since its details will be mapped to the Rancher local principal account. Make sure that this level of privilege is appropriate before you continue. 
```

    | Application Secret | Key Value                             |
    | Endpoint           | https://login.microsoftonline.com/    |

    **For Custom Endpoints:**

>     The following table maps the custom config values you copied in the Azure portal to the fields in Rancher:

Reviewer: ```suggestion
    The following table maps your custom config values to Rancher fields:
```

    | Rancher Field      | Azure Value                           |
    | ------------------ | ------------------------------------- |
    | Graph Endpoint     | Microsoft Graph API Endpoint          |
    | Token Endpoint     | OAuth 2.0 Token Endpoint              |
    | Auth Endpoint      | OAuth 2.0 Authorization Endpoint      |

>     **Important:** When entering the Graph Endpoint in a custom config, remove the tenant ID from the URL, like below:

Reviewer: ```suggestion
    **Important:** When entering the Graph Endpoint in a custom config, remove the tenant ID from the URL:
```
